apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/JobCreationOptions.cs

```diff
@@ -37,5 +37,11 @@ public abstract class JobCreationOptions
         /// and <see cref="JobIdPrefix"/>.
         /// </summary>
         public string JobId { get; set; }
+
+        /// <summary>
+        /// If specified, the job will be created in this location. Otherwise, it will be created in the client's
+        /// default location.
+        /// </summary>
```

Comment:
Add a `<seealso>` to `WithDefaultLocation`?

Reply:
Done - and also added comments to CreateDatasetOptions.